src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/EventReceiverStatusPoller.kt

```diff
@@ -0,0 +1,41 @@
+package it.pagopa.ecommerce.eventdispatcher.services
+
+import it.pagopa.ecommerce.eventdispatcher.config.RedisStreamEventControllerConfigs
+import it.pagopa.ecommerce.eventdispatcher.config.redis.EventDispatcherReceiverStatusTemplateWrapper
+import it.pagopa.ecommerce.eventdispatcher.config.redis.bean.ReceiversStatus
+import java.time.OffsetDateTime
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+/**
+ * Poller class that taken into account receiver status polling. Statuses are polled as per
+ * eventController.status.pollingChron parameter value and results updated into Redis document
+ * uniquely identified by this module instance
+ */
+@Component
+class EventReceiverStatusPoller(
+  @Autowired
+  private val eventDispatcherReceiverStatusTemplateWrapper:
+    EventDispatcherReceiverStatusTemplateWrapper,
+  @Autowired private val inboundChannelAdapterHandlerService: InboundChannelAdapterHandlerService,
+  @Autowired private val redisStreamEventControllerConfigs: RedisStreamEventControllerConfigs
+) {
+
+  private val logger = LoggerFactory.getLogger(javaClass)
+
+  @Scheduled(cron = "\${eventController.status.pollingChron}")
```

Comment:
Polling defeats a bit the purpose of Pub/Sub, we should consider refactoring this to use Spring Data Redis' [`StreamReceiver`](https://docs.spring.io/spring-data/redis/reference/redis/redis-streams.html#reactive-streamreceiver) later on

Reply:
This polling class is used to update each event dispatcher instance event receivers statuses